microsoft-edge/webview2/samples/webview2browser.md

```diff
@@ -37,27 +37,21 @@ This sample is built as a Win32 [Visual Studio 2019](https://visualstudio.micros
 
 
 <!-- ====================================================================== -->
-## Step 1: Install a preview channel of Microsoft Edge
-
-*  If not installed yet, install a preview channel of Microsoft Edge on a supported operating system.  To do this, go to [Become a Microsoft Edge Insider](https://www.microsoft.com/edge/download/insider/).
-
-
-<!-- ====================================================================== -->
-## Step 2: Install Visual Studio
+## Step 1: Install Visual Studio
 
 1. Install [Visual Studio](https://visualstudio.microsoft.com/vs/), including C++ support.
 
-The [WebView2 Runtime](https://developer.microsoft.com/microsoft-edge/webview2/) is recommended for the installation.<!-- TODO: give actionable steps/instructions.  Why preview channel and also Runtime? -->  The minimum version is 86.0.622.38.
+The [WebView2 Runtime](https://developer.microsoft.com/microsoft-edge/webview2/) is recommended for the installation.<!-- TODO: give actionable steps/instructions -->  The minimum version is 86.0.622.38.
```

Comment:
I think we can remove this line. The other sample apps don't have this line, and most Windows versions will now come with WebView2 installed

Reply:
thx, was wondering, removed